reference/UiTPAS.v2.json

```diff
@@ -1478,6 +1478,108 @@
         ]
       }
     },
+    "/passholders/membership-options": {
+      "get": {
+        "summary": "Retrieve UiTPAS membership options",
```

Comment:
Nit: Haal misschien "UiTPAS" uit de summary, want dit maakt ze nogal lang waardoor het wordt afgekort naar "Retrieve UiTPAS membership optio..." in de sidebar in Stoplight, en UiTPAS is misschien wat redundant aangezien het de UiTPAS API is?